### PR TITLE
Refactor composer view with new container view

### PR DESCRIPTION
**Pull request by nuno-vieira**
# In this PR
- Refactors the composer view to use the new container stack view.

**Note:** Right now the composer view controller needs to know about the implementation details of ComposerView in order to use it. This will change in the next PR.

### CODE REVIEW

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift [hunk 1]
```diff
@@ -289,7 +290,10 @@ public class ContainerStackView: UIView {
 
         Animate(isAnimated: animated) {
             subview.alpha = 0
-            self.hideConstraintsByView[subview]?.width.isActive = true
+
+            if self.axis == .horizontal {
+                self.hideConstraintsByView[subview]?.width.isActive = true
+            }
             self.hideConstraintsByView[subview]?.height.isActive = true
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/ContainerStackView.swift:297]: Isn't there a missing `else` there?

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift [hunk 2]
```diff
@@ -298,8 +302,9 @@ public class ContainerStackView: UIView {
         }
     }
 
-    func showSubview(_ subview: UIView, animated: Bool = true) {
-        assert(subviews.contains(subview))
+    public func showSubview(_ subview: UIView, animated: Bool = true) {
+        guard subviews.contains(subview) else { return }
+        guard isHidden(subview) else { return }
```
Comment (VojtaStavik) [inline Sources/StreamChatUI/CommonViews/ContainerStackView.swift:307]: tbh: I'd find doing `guard subview.alpha == 0 else { return }` less confusing than this helper.

File: iMessageClone/iMessageChatMessageComposerView.swift [hunk 3]
```diff
@@ -10,31 +10,58 @@ final class iMessageChatMessageComposerView: ChatMessageComposerView {
...
+
+        centerContentContainer.layer.cornerRadius = 18
         messageInputView.textView.font = .systemFont(ofSize: 15)
         
         attachmentButton.setImage(
             UIImage(systemName: "camera.fill"),
             for: .normal
         )
+        attachmentButton.widthAnchor.constraint(equalToConstant: 30).isActive = true
+        attachmentButton.widthAnchor.constraint(equalToConstant: 30).isActive = true
```
Comment (VojtaStavik) [inline iMessageClone/iMessageChatMessageComposerView.swift:65]: same line twice

**Review by VojtaStavik — APPROVED** [review]
OK 👍 Let's deal with the details in a follow up PR

File: Sources/StreamChatUI/CommonViews/ContainerStackView.swift (hunk 1 above)
Comment (nuno-vieira) [reply Sources/StreamChatUI/CommonViews/ContainerStackView.swift:297]: No, it is fine like this, only if horizontal we want to activate it, but if not, we don't need to anything more 👍

**Pull request merged**